changelog.txt

```diff
@@ -1,7 +1,7 @@
 *** WooCommerce Shipping & Tax Changelog ***
 
 = 1.25.8 - 2021-xx-xx =
-* Tweak - Display better errors on checkout page when address fields are missing / invalid.
+* Tweak - Add support for new jetpack data connection.
```

Comment:
```suggestion
* Tweak - Add support for new Jetpack 9.5 data connection.
```

I think it would be good to somehow mention Jetpack 9.5 specifically so people know which version exactly requires this.